test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,66 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const md5 = require('md5');
+
+const path = 'https://api.github.com/users/aperdomob';
+const path2 = 'https://api.github.com/repos/aperdomob';
+
+const query1 = {
```

Comment:
Try to use more descriptive names (Instead of query1, something like userQuery for example. Same for the other 2).

Reply:
Solved